### PR TITLE
Fix persimmon `n_rot` conversion in `convert-persimmon-to-gguf.py`

### PR DESCRIPTION
**Pull request by lx200916**
Fix `rope_dimension_count` param value in `convert-persimmon-to-gguf.py` according to those in `convert-hf-to-gguf.py`

### CODE REVIEW

**Comment by lx200916** [comment]
Seems like something wrong with CI itself, it says `ERROR: Could not install packages due to an OSError: [Errno 28] No space left on device`.

File: convert-persimmon-to-gguf.py [hunk 1]
```diff
@@ -88,7 +88,8 @@ def main():
     gguf_writer.add_embedding_length(hidden_size)
     gguf_writer.add_block_count(block_count)
     gguf_writer.add_feed_forward_length(hparams.ffn_hidden_size)
-    gguf_writer.add_rope_dimension_count(hidden_size // head_count)
+    # ref: https://github.com/ggerganov/llama.cpp/pull/4889/commits/eea19039fc52ea2dbd1aab45b59ab4e3e29a3443
+    gguf_writer.add_rope_dimension_count(hidden_size // head_count //2)
```
Comment (ggerganov) [inline convert-persimmon-to-gguf.py:92]:
```suggestion
    gguf_writer.add_rope_dimension_count(hidden_size // head_count // 2)
```

**Review by ggerganov — APPROVED** [review]
Btw, is this script (`convert-persimmon-to-gguf.py`) still needed since `convert-hf-to-gguf.py` already supports Persimmon?

**Comment by lx200916** [comment]
> Btw, is this script (`convert-persimmon-to-gguf.py`) still needed since `convert-hf-to-gguf.py` already supports Persimmon?

😂Initially, I tried to use the convert.py with shell commands from the readme to convert the persimmon weights from hf, but an exception was thrown. I suspect that it might be necessary to use convert-persimmon-to-gguf.py along with the official weights, and then this issue occurred. Perhaps we need clearer instructions about these convert.py scripts?

**Pull request merged**